torch/csrc/autograd/utils/wrap_outputs.h

```diff
@@ -58,6 +59,10 @@ inline PyObject* wrap(at::Scalar scalar) {
   return wrap(make_variable(scalar_to_tensor(scalar)));
 }
 
+inline PyObject* wrap(at::QScheme qscheme) {
+  return wrap(at::Scalar(static_cast<int>(qscheme)));
```

Comment:
if you have PyObjects representing these QSchemes I don't know why you'd want to return a scalar.  Can you do the same as with THPDtype?